client/errors.go

```diff
@@ -4,9 +4,30 @@ import (
 	"fmt"
 )
 
+var (
```

Comment:
i would not keep parenthesis until i know that it gonna be expanded soon (say, next commit). but it is cosmetic

Reply:
I would left it as is, if you don't mind